onebusaway-android/src/main/java/org/onebusaway/android/ui/MaterialListItem.java

```diff
@@ -13,24 +13,31 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
-package org.onebusaway.android.report.ui.model;
+package org.onebusaway.android.ui;
 
 /**
  * Model for report type
```

Comment:
Could you please update the comment to reflect that this is a more generic list item with material design now?